packages/tasks/src/snippets/js.ts

```diff
@@ -41,6 +41,35 @@ for await (const chunk of inference.chatCompletionStream({
 		return snippetBasic(model, accessToken);
 	}
 };
+
+export const snippetImageTextToTextGeneration = (model: ModelDataMinimal, accessToken: string): string => {
+	if (model.tags.includes("conversational")) {
+		// Conversational model detected, so we display a code snippet that features the Messages API
+		return `import { HfInference } from "@huggingface/inference";
+
+const inference = new HfInference("${accessToken || `{API_TOKEN}`}");
+const image_url = "https://cdn.britannica.com/61/93061-050-99147DCE/Statue-of-Liberty-Island-New-York-Bay.jpg"
```

Comment:
nit: but in js, we usually prefer camelCase `imagUrl` rather than snake_case `image_url`. No?

Reply:
addressed in s/f6d7ff8a2be24abe60ded03ee4e7ec4eb2ed33f8